src/Form/FilePersisterServiceSettingsForm.php

```diff
@@ -306,9 +317,11 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       ->save();
     $this->config('strawberryfield.storage_settings')
       ->set('file_scheme', $form_state->getValue('file_scheme'))
+      ->set('file_path', trim($form_state->getValue('file_path')," \n\r\t\v\0/"))
```

Comment:
Lovely! But I think for this case we need to make sure these paths are sanitized. Also S3 has size limits. Look at the method where I refactored the hook. I have a few notes there about this

Reply:
So checkPlain and limit the path to 128 characters? Ok to just set the field character limit?

Reply:
I think we need to make folder validation method. Should not have spaces. Should have a controlled list of characters (like no utf8 strawberries).  Here is a list of what needs to be checked. I agree we are also allowing local storage, e.g what @giancarlobi does, but still not sure I want to have on a local unix filesystem something that fights a 1986 unix spec too much. See https://www.ezs3.com/public/232.cfm

Reply:
Ok, I'll work on a validation method.